docs/Tutorials/Kiln.md

```diff
@@ -0,0 +1,96 @@
+---
+Description: How to run Besu and Teku on Kiln
+---
+
+# Test Besu and Teku on Kiln
+
+You can test Besu as an [execution client](../Concepts/Merge.md) and [Teku](https://docs.teku.consensys.net/en/stable/)
+as a [consensus client](../Concepts/Merge.md) on the [Kiln testnet](https://blog.ethereum.org/2022/03/14/kiln-merge-testnet/).
+
+## Prerequisites
+
+1. Install [Besu](../HowTo/Get-Started/Installation-Options/Options.md) and
+   [Teku](https://docs.teku.consensys.net/en/stable/HowTo/Get-Started/Installation-Options/Install-Binaries/).
+
+1. Download the [configuration files](https://github.com/eth-clients/merge-testnets/tree/main/kiln):
+
+    ```bash
+    git clone https://github.com/eth-clients/merge-testnets.git
+    cd merge-testnets/kiln
+    ```
+
+1. Generate the JWT secret:
+
+    ```bash
+    openssl rand -hex 32 | tr -d "\n" > jwtsecret.hex
+    ```
+
+    You must pass this file to both Besu and Teku.
+
+## Start Besu
+
+Run the following command:
+
+```bash
+besu \
+  --network=kiln              \
+  --rpc-http-enabled=true     \
+  --rpc-http-host="0.0.0.0"   \
+  --rpc-http-cors-origins="*" \
+  --rpc-ws-enabled=true       \
+  --rpc-ws-host="0.0.0.0"     \
+  --host-allowlist="*"        \
+  --engine-host-allowlist="*" \
+  --engine-jwt-enabled=true   \
+  --engine-rpc-http-port=8550 \
+  --engine-rpc-ws-port=8551   \
+  --Xmerge-support=true       \
+  --engine-jwt-secret=<path_to_jwtsecret>
+```
+
+See the [`--engine-*`](../Reference/CLI/CLI-Syntax.md#engine-host-allowlist) CLI options for more information on running
+Besu as an execution client.
+
+## Start Teku
+
+Run the following command:
+
+```bash
+teku \
+  --data-path "datadir-teku"               \
+  --network kiln                           \
+  --p2p-discovery-bootnodes "enr:-Iq4QMCTfIMXnow27baRUb35Q8iiFHSIDBJh6hQM5Axohhf4b6Kr_cOCu0htQ5WvVqKvFgY28893DHAg8gnBAXsAVqmGAX53x8JggmlkgnY0gmlwhLKAlv6Jc2VjcDI1NmsxoQK6S-Cii_KmfFdUJL2TANL3ksaKUnNXvTCv1tLwXs0QgIN1ZHCCIyk" \
+  --ee-endpoint http://localhost:8550      \
+  --Xee-version kilnv2                     \
+  --ee-jwt-secret-file <path_to_jwtsecret> \
+  --log-destination console                \
+  --validators-proposer-default-fee-recipient=<eth_address_of_fee_recipient>
```

Comment:
Do we have any documentation about what address to add for `<eth_address_of_fee_recipient>`. If not, then there's probably a gap in the doco.